### ISSUE TITLE
sentence-per-line false positive with e.g. and i.e.

### ISSUE THREAD

**Issue by davidhewitt** [rvben/rumdl#422]
### rumdl version

0.1.18

### Installation method

Pre-built binary

### Bug description

It seems that `i.e.` and `e.g.` are incorrectly treated as sentence ends by `rumdl` in `sentence-per-line` mode.

### Steps to reproduce

```markdown
# Hi

In addition, not all platforms (e.g. Wasm) are supported by `inventory`, which is used in the implementation of the feature.

This `#[cfg]` marks code that is only available when building for the unlimited Python API (i.e. PyO3's `abi3` feature is not enabled).
```

### Expected behavior

File lints without complaint.

### Actual behavior

```
$ uvx rumdl check test.md
test.md:3:1: [MD013] Line contains 2 sentences (one sentence per line required) [*]
test.md:5:1: [MD013] Line contains 2 sentences (one sentence per line required) [*]
```

### Configuration

```toml
[tool.rumdl.MD004]
style = "dash"

[tool.rumdl.MD013]
line_length = 0
code_blocks = false
tables = false
headings = false
reflow = true
reflow_mode = "sentence-per-line"
```

### Operating system

macOS

**Comment by rvben** [comment]
Thanks @davidhewitt for this feedback, those false positives are annoying. I expect the fix for this to land in 0.1.19.

**Comment by davidhewitt** [comment]
Thanks, confirmed fixed in https://github.com/PyO3/pyo3/pull/5805 !

**Comment by rvben** [comment]
Thanks for verifying, @davidhewitt ! 🚀
